portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -64,4 +78,43 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     String json = new Gson().toJson(commentList);
     response.getWriter().println(json);
   }
+
+    private String getUsername(String id) {
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        Query query =
+            new Query("user")
+                .setFilter(new Query.FilterPredicate("id", Query.FilterOperator.EQUAL, id));
+        PreparedQuery results = datastore.prepare(query);
+        Entity entity = results.asSingleEntity();
+
+        if (entity == null) {
+            return null;
+        }
+
+        System.out.println("======NOT+NULLLLLLL=======");
+
+
+        String nickname = (String) entity.getProperty("username");
+        return nickname;
+  }
+
+  @Override
+  public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    boolean deleteAll = true;
+    if (deleteAll) {
```

Comment:
What's going on here?  It looks like deleteAll is happening no matter what...
If the intent is to only implement the deleteAll functionality in this PR and come back in a separate PR and support individual deletions, I'd just leave the boolean and if statement out of the code for this round and leave a comment saying TODO: Implement individual comment deletion.

Reply:
Resolved and also learned that you cannot pass a body into a delete method on the server

Reply:
Ah right, that sounds familiar.  I think any parameters need to go in the URL instead for delete!